modules/platforms/dotnet/Apache.Ignite/Apache.Ignite.csproj

```diff
@@ -38,6 +38,7 @@
     <PackageReference Include="NodaTime" Version="[3.*,)" />
     <PackageReference Include="Remotion.Linq" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[6.*,)" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="[8.*,)" />
```

Comment:
Initially I planned to avoid this extra dependency in the core package by moving the extensions to a separate package (`Apache.Ignite.DependencyInjection` or something like that).

However, [Microsoft.Extensions.DependencyInjection.Abstractions](https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection.Abstractions) is small (~235 KB) and does not have any dependencies, so I guess we can keep this here.

Thoughts?

Reply:
Example: https://www.nuget.org/packages/Npgsql.DependencyInjection

Reply:
My original thought was the same, but arguably there is not much benefit from it.

Reply:
Ok, let's keep it in the same assembly/package.